httpx/dispatch/connection_pool.py

```diff
@@ -128,14 +128,7 @@ def num_connections(self) -> int:
         return response
 
     async def acquire_connection(self, origin: Origin) -> HTTPConnection:
-        logger.debug(f"acquire_connection origin={origin!r}")
-        connection = self.active_connections.pop_by_origin(origin, http2_only=True)
-        if connection is None:
-            connection = self.keepalive_connections.pop_by_origin(origin)
-
-        if connection is not None and connection.is_connection_dropped():
-            self.max_connections.release()
-            connection = None
+        connection = self.pop_connection(origin)
```

Comment:
It seems we've lost the `logger.debug()` call here?

Reply:
Yep, will restore!